app/assets/stylesheets/app/profile.scss

```diff
@@ -109,19 +109,51 @@
     padding: 2% 0;
   }
 
+  &__number-container {
+    align-items: center;
+    display: flex;
+    flex: 1;
+    flex-direction: column;
+    justify-content: center;
+
+    &:not(:last-child) {
+      border-right: 1px solid $light-border;
+    }
+  }
+
+  &__number {
+    align-items: flex-end;
+    display: flex;
+    flex: 1;
+    font-size: 150%;
+
+    &--positive {
+      color: #0f0;
+    }
+
+    &--zero {
+      color: $orange;
+    }
+
+    &--negative {
+      color: #f00;
```

Comment:
Color literals like `#f00` should only be used in variable declarations; they should be referred to via variable everywhere else.